_data-prepper/pipelines/configuration/processors/key-value.md

```diff
@@ -10,20 +10,20 @@ nav_order: 45
 
 ## Overview
 
-The `key_value` processor parses a field into key/value pairs. The following table describes `key_value` processor options available that help you parse field information into pairs.
-
-Option | Required | Type | Description
-:--- | :--- | :--- | :---
-source | No | String | The key in the event that is parsed. Default value is `message`.
-destination | No | String | The destination key for the parsed source output. Outputting the parsed source overwrites the value of the key if it already exists. Default value is `parsed_message`
-field_delimiter_regex | Conditionally | String | A regex specifying the delimiter between key/value pairs. Special regex characters such as `[` and `]` must be escaped using `\\`. This option cannot be defined at the same time as `field_split_characters`.
-field_split_characters | Conditionally | String | A string of characters to split between key/value pairs. Special regex characters such as `[` and `]` must be escaped using `\\`. Default value is `&`. This option cannot be defined at the same time as `field_delimiter_regex`.
-key_value_delimiter_regex| Conditionally | String | A regex specifying the delimiter between a key and a value. Special regex characters such as `[` and `]` must be escaped using `\\`. There is no default value. This option cannot be defined at the same time as `value_split_characters`.
-value_split_characters | Conditionally | String | A string of characters to split between keys and values. Special regex characters such as `[` and `]` must be escaped using `\\`. Default value is `=`. This option cannot be defined at the same time as `key_value_delimiter_regex`.
-non_match_value | No | String | When a key/value cannot be successfully split, the key/value is placed in the `key` field, and the specified value is placed in the value field. Default value is `null`.
-prefix | No | String | A prefix given to all keys. Default value is empty string.
-delete_key_regex | No | String | A regex used to delete characters from the key. Special regex characters such as `[` and `]` must be escaped using `\\`. There is no default value.
-delete_value_regex | No | String | A regex used to delete characters from the value. Special regex characters such as `[` and `]` must be escaped using `\\`. There is no default value.
+You can use the `key_value` processor to parse the specified field into key-value pairs. You can customize the `key_value` processor to parse field information with the following options.
+
+| Option | Required | Type | Default value | Description | Example |
+| :--- | :--- | :--- | :--- | :--- |
+| source | No | String | `message` | The key in the event that is parsed. | `source` is `"message1"`. `{"message1": {"key1=value1"}, "message2": {"key2=value2"}}` parses into `{"message1": {"key1=value1"}, "message2": {"key2=value2"}, "parsed_message": {"key1": "value1"}}`. |
```

Comment:
```suggestion
| source | No | String | `message` | The message field to be parsed.  | `source` is `"message1"`. `{"message1": {"key1=value1"}, "message2": {"key2=value2"}}` parses into `{"message1": {"key1=value1"}, "message2": {"key2=value2"}, "parsed_message": {"key1": "value1"}}`. |
```

Reply:
When you remove the "type", "required", and "default value" columns, see if the examples are formatted nicely in one line. If they're not, I suggest breaking them out into a section below the table. In any case, make the examples complete sentences.